### ISSUE TITLE
Reviewing of User Scenarios

### ISSUE THREAD

**Issue by Elifnurdeniz** [bounswe/bounswe2024group6#293]
## Issue Description
The user scenarios were written and added to wiki in the past weeks. We want to see if we can improve them. The assignee will check them for improvements.

## Tasks

- [x] Checking User Scenarios

## Estimated Time

**Estimated Time:** 1 hours

## Deadline

**Deadline:** 15/10/2024- 15:00

## Reviewer Assignment

**Reviewer:** @yunusemreozdemir 
**Review Deadline:** 15/10/2024- 15:00

**Comment by Elifnurdeniz** [comment]
I edited some related requirements under [User Scenario 3](https://github.com/bounswe/bounswe2024group6/wiki/User-Scenario-3) and [User Scenario 4](https://github.com/bounswe/bounswe2024group6/wiki/User-Scenario-4). I added the requirements that defines inclusive actions of some user roles. We decided to use mockups instead of drawing in the illustration part. We will ask if we are allowed to make changes and then I will edit them too.
